relay-server/src/services/processor.rs

```diff
@@ -2134,22 +2135,22 @@ impl EnvelopeProcessorService {
             rate_limits,
             data,
             project_key,
-            start_time,
+            received_at,
             sent_at,
             source,
         } = message;
 
-        let received_timestamp = UnixTimestamp::from_instant(start_time);
+        let received_timestamp =
```

Comment:
I deemed this as a safe operation.

Reply:
What do you mean by this? Why wouldn't it be safe?

Reply:
By safe I meant semantically, meaning if it makes sense to default to now.

Reply:
Strictly speaking, defaulting to 1970-01-01 (`UnixTimestamp::from_secs(0)`) would make sense semantically, because `from_datetime` returns `None` only if the datetime is less than that. Either way it's a programming error or data corruption if it happens, since we control `received_at` ourselves.